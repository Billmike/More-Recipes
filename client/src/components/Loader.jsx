import React from 'react';

const Loader = () => (
  <div className="ring">
    <p className="loader-text">Loading</p>
    <span className="loader-span"></span>
  </div>
);

export default Loader;
