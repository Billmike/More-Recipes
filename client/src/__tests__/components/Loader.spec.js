import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../components/Loader';

describe('<Loader />', () => {
  it('Should mount the loader component', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
