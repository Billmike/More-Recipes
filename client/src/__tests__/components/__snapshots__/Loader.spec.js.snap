// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Should mount the loader component 1`] = `
<div
  className="ring"
>
  <p
    className="loader-text"
  >
    Loading
  </p>
  <span
    className="loader-span"
  />
</div>
`;
